draft-ietf-quic-recovery.md

```diff
@@ -653,27 +698,8 @@ follows:
 
 ### On Sending a Packet
 
-After any packet is sent, be it a new transmission or a rebundled transmission,
-the following OnPacketSent function is called.  The parameters to OnPacketSent
-are as follows:
-
-* packet_number: The packet number of the sent packet.
-
-* retransmittable: A boolean that indicates whether a packet is
-  retransmittable. If true, it is expected that an acknowledgement will
-  be received, though the peer could delay sending the ACK frame containing
-  it by up to the MaxAckDelay.
-
-* in_flight: A boolean that indicates whether the packet counts towards bytes in
-  flight.
-
-* is_crypto_packet: A boolean that indicates whether the packet contains
-  cryptographic handshake messages critical to the completion of the QUIC
-  handshake. In this version of QUIC, this includes any packet with the long
-  header that includes a CRYPTO frame.
-
-* sent_bytes: The number of bytes sent in the packet, not including UDP or IP
-  overhead, but including QUIC framing overhead.
+After any packet is sent.  The parameters to OnPacketSent are described in detail
+above in {{sent-packets-fields}}.
```

Comment:
I believe the error is because `detail` needs to wrap to the next line.
```
After any packet is sent.  The parameters to OnPacketSent are described in
detail above in {{sent-packets-fields}}.
```

Reply:
Thanks Nick, fixed.